samples/expert-finder-js/README.md

```diff
@@ -0,0 +1,92 @@
+---
+page_type: sample
+description: This sample demonstrates how to use a copilot plugin that can be used to search candidates for job. This sample can also be used as a search based messaging extension.
+products:
+- office-teams
+- copilot-m365
+languages:
+- javascript
+---
+
+# Expert Finder Copilot Plugin
```

Comment:
Please refer this: 
![image](https://github.com/user-attachments/assets/5dcc0ba5-b027-43b6-9623-05fc43951000)
https://github.com/OfficeDev/Copilot-for-M365-Samples/tree/main/samples/msgext-northwind-inventory-action-ts

Reply:
changed